src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+import { themeGet, position } from 'styled-system';
+import { triangle } from 'polished';
+import { Box, NakedButton, Icon } from '../';
+
+const pointerWidth = '16px';
+const pointerHeight = '8px';
```

Comment:
We should probably use `rem`s, right?

```suggestion
import { rem, triangle } from 'polished';
import { Box, NakedButton, Icon } from '../';

const pointerWidth = rem(16);
const pointerHeight = rem(8);
```

Reply:
👍